app/pages/UcsfServiceTypePage/UcsfServiceTypePage.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '~styles/utils/_helpers.scss';
+
+.eligibilityPage {
```

Comment:
Nit: Having the word "eligibility" in all of these class names is a little bit confusing, since this is not the eligibilities page.

That said, it might be better to do a followup refactoring where we actually share common CSS/React components for the stuff that is the same between the different UCSF pages.